charts/argo-services/templates/executor-default-rolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: executor-default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: executor
+subjects:
+  - kind: ServiceAccount
+    name: default
```

Comment:
I guess it's ok to bind this to the default SA (maybe not 100% ideal since there's stuff other than Argo in this namespace but create/patch workflowtaskresults doesn't seem like a big deal so 👍)

Reply:
Yeah - probably not ideal to be using the default SA for workflows anyways. But didn't wanna tackle that now, particular if Argo Workflows might go anyways

Reply:
💯